src/components/form/file-upload/file-upload.tsx

```diff
@@ -139,16 +140,17 @@ export const FileUpload: FC<FileUploadProps> = ({
                     {labelFileSize}
                 </Paragraph>
             </section>
-            // TODO: use ours (if possible)
-            <input
+            <Input
```

Comment:
I checked it on storybook and it doesn't work with ours, since there is a div with a frame - better use the normal `<input />`